app/src/main/java/woowacourse/omok/view/NewRoomActivity.kt

```diff
@@ -0,0 +1,61 @@
+package woowacourse.omok.view
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Button
+import android.widget.EditText
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.widget.addTextChangedListener
+import woowacourse.omok.R
+import woowacourse.omok.model.database.GameRoomDao
+import woowacourse.omok.model.database.Room
+
+class NewRoomActivity : AppCompatActivity() {
+    private lateinit var startButton: Button
+    private lateinit var gameNameInput: EditText
+    private lateinit var gameRoomDao: GameRoomDao
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_new_room)
+
+        gameRoomDao = GameRoomDao(this)
+        initializeGameNameInput()
+        initializeStartButton()
+    }
+
+    private fun initializeGameNameInput() {
+        gameNameInput =
+            findViewById<EditText>(R.id.et_new_game).apply {
+                addTextChangedListener(
+                    onTextChanged = { currentText, _, _, _ ->
+                        startButton.isClickable = currentText?.length in RANGE_ROOM_NAME_LENGTH
+                    },
+                )
+            }
```

Comment:
확장함수를 사용하지 않더라도 괜찮은 가독성을 가져가 볼 수도 있답니다 :)
들여쓰기를 줄이는 건 덤이구요!
```suggestion
        gameNameInput = findViewById<EditText>(R.id.et_new_game)
        gameNameInput.addTextChangedListener(
            onTextChanged = { currentText, _, _, _ ->
                startButton.isClickable = currentText?.length in RANGE_ROOM_NAME_LENGTH
            },
        )
    }
```